blocks/link-group/link-group.css

```diff
@@ -41,7 +41,7 @@
 }
 
 .block.link-group.active .icon-slim-arrow-right {
-  transform: rotate(90deg);
+  transform: rotate(-90deg);
```

Comment:
[See this conversation](https://adobe-dx-support.slack.com/archives/C06H8UNQ3RS/p1707762810730929?thread_ts=1707236316.695689&cid=C06H8UNQ3RS)